packages/opentelemetry-sdk-trace-base/src/config.ts

```diff
@@ -98,7 +98,9 @@ export function buildSamplerFromEnv(): Sampler {
       diag.error(
         `OTEL_TRACES_SAMPLER value "${sampler}" invalid, defaulting to "${FALLBACK_OTEL_TRACES_SAMPLER}".`
```

Comment:
```suggestion
        `OTEL_TRACES_SAMPLER value "${sampler}" invalid, defaulting to "${TracesSamplerValues.ParentBasedAlwaysOn}".`
```

And then remove the `const FALLBACK_OTEL_TRACES_SAMPLER` above.

Reply:
good point, addressed in [9c244a7](https://github.com/open-telemetry/opentelemetry-js/pull/5494/commits/9c244a79f92ea1606095fde2c03ac6756287e8b4) 👍